contracts/test/vault/oeth-swapper.js

```diff
@@ -82,27 +84,44 @@ describe("OETH Vault - Swapper", () => {
       }
     });
 
+    it("Should allow the governor to change allowed swap undervalue", async () => {
+      const { governor, oethVault } = fixture;
+
+      const tx = oethVault.connect(governor).setSwapAllowedUndervalue(10);
+
+      await expect(tx)
+        .to.emit(oethVault, "SwapAllowedUndervalueChanged")
+        .withArgs(10);
+
+      expect(await oethVault.allowedSwapUndervalue()).to.equal(10);
+    });
+
+    it("Should not allow anyone else to set allowed swap undervalue", async () => {
+      const { strategist, josh, oethVault } = fixture;
+
+      for (const user of [strategist, josh]) {
```

Comment:
I like this style of testing it with multiple user, and including negative  testing the strategist.

Reply:
credit to @shahthepro for this